django/db/backends/postgresql/operations.py

```diff
@@ -271,5 +274,18 @@ def explain_query_prefix(self, format=None, **options):
             prefix += ' (%s)' % ', '.join('%s %s' % i for i in extra.items())
         return prefix
 
-    def ignore_conflicts_suffix_sql(self, ignore_conflicts=None):
-        return 'ON CONFLICT DO NOTHING' if ignore_conflicts else super().ignore_conflicts_suffix_sql(ignore_conflicts)
+    def conflicts_suffix_sql(
+        self, opts, fields, on_conflicts=ON_CONFLICTS_NONE, update_fields=[], unique_fields=[]
+    ):
+        if on_conflicts == ON_CONFLICTS_IGNORE:
+            return 'ON CONFLICT DO NOTHING'
+
+        elif on_conflicts == ON_CONFLICTS_UPDATE:
+            result = 'ON CONFLICT(%s) DO UPDATE SET ' % (', '.join(unique_fields))
```

Comment:
I feel like this will break when the field is a reserved keyword (also for all other backends). At the minimum we should also have tests that will show that reserved fields will work as expected (ie are quoted properly)

Reply:
Hi @apollo13,
I cannot get what you said.
Could you explain it more detailed?
Thanks.

Reply:
Thanks, I got the problem you mentioned.